extensions/sql-database-projects/src/test/dialogs/utils.test.ts

```diff
@@ -1,32 +0,0 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the Source EULA. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-import * as should from 'should';
-import * as constants from '../../common/constants';
-import { SqlTargetPlatform } from 'sqldbproj';
-import { getDefaultDockerImageWithTag, getDockerBaseImages } from '../../dialogs/utils';
-
-describe('Tests to verify dialog utils functions', function (): void {
```

Comment:
this file shouldn't be deleted, right?

Reply:
I have these tests added in data-workspace extension, hence removed this file from here.

Reply:
There were some docker tests in this file that shouldn't be deleted

Reply:
Sorry, my bad. I forgot there were tests up as well, I will place it back.